docs/fast_data/monitoring/dashboards/single_views.md

```diff
@@ -4,64 +4,17 @@ title: Single Views
 sidebar_label: Single Views
 ---
 
-This Dashboard is mainly concerned with Single Views related concepts and as such, it needs the Single View Creator and/or Single View Trigger Generator to expose the metrics it displays.
+This dashboard focuses on the [Single View Trigger Generator](/fast_data/single_view_trigger_generator.md) and the [Single View Creator](/fast_data/single_view_creator.md) services. It displays data related to strategies execution, aggregation, Single Views and Projection Changes, as well as metrics of the services hardware performance. Among the displayed features you can find:
 
-:::caution
-Make sure the services you need the metrics from have the "Enable metric gathering" flag enabled in the console.
-:::
+- Strategy execution time
+- Single View creation time
+- Number of Single View creation error by type
+- Time from Projection to Single View
+- Number of Single View Creator instances
+- ....
 
-Click <a download target="_blank" href="/docs_files_to_download/single-view-creator.json">**here**</a> to download the dashboard.
-
-### Single View Creator Service
-
-When working with Fast Data, you should already have one or more Single View Creator services, which will expose the relevant metrics. No additional configuration is needed.
-
-The metrics exposed by the Single View Creator Service are:
-
-| Metric Name                        | Description                                                                                       | Buckets (ms)                                                     | Available since version |
-| ---------------------------------- | ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- | ----------------------- |
-| svc_pcs_handled_total              | counts how many projections changes were successfully processed                                   |                                                                  | 3.0.0                   |
```

Comment:
That's a very good point that I totally missed, I will do a tour now filling all dashboards descriptions and updating the linked files

Reply:
I made a quick tour of the dashbaords and some have description some others are enough with the title, like "Message consumption rate" should be enough explanation. Since adding a description would be very time consuming and possibily pointless for some widgets I'd proceed as we are and listen to feedback if something is not properly clear. wdyt?

Reply:
Maybe we should dedicate a specific section in plugins documentation where it is explained which metrics each service exposes.